tools/make_hgrid/make_hgrid.c

```diff
@@ -842,107 +840,108 @@ int main(int argc, char* argv[])
     }
   }
   else if( my_grid_type == SIMPLE_CARTESIAN_GRID ) {
     strcpy(geometry, "planar");
     strcpy(north_pole_tile, "none");
     if(nxbnds1 != 2 || nybnds1 != 2 ) mpp_error("make_hgrid: grid type is 'simple_cartesian_grid', number entry entered "
-						"through --xbnds and --ybnds should be 2");
+                                                "through --xbnds and --ybnds should be 2");
     if(nxbnds2 != 1 || nybnds2 != 1 ) mpp_error("make_hgrid: grid type is 'simple_cartesian_grid', number entry entered "
-						"through --nlon and --nlat should be 1");
+                                                "through --nlon and --nlat should be 1");
     if(simple_dx == 0 || simple_dy == 0) mpp_error("make_hgrid: grid_type is 'simple_cartesian_grid', "
-						   "both simple_dx and simple_dy both should be specified");
+                                                   "both simple_dx and simple_dy both should be specified");
   }
   else if( my_grid_type == SPECTRAL_GRID ) {
     if(nxbnds2 != 1 || nybnds2 != 1 ) mpp_error("make_hgrid: grid type is 'spectral_grid', number entry entered "
-						"through --nlon and --nlat should be 1");    
+                                                "through --nlon and --nlat should be 1");
   }
   else if( my_grid_type == CONFORMAL_CUBIC_GRID ){
     strcpy(projection, "cube_gnomonic");
     strcpy(conformal, "FALSE");
     if(nxbnds2 != 1 ) mpp_error("make_hgrid: grid type is 'conformal_cubic_grid', number entry entered "
-				"through --nlon should be 1");
+                                "through --nlon should be 1");
     if(nratio < 1) mpp_error("make_hgrid: grid type is 'conformal_cubic_grid', nratio should be a positive integer");
   }
   else if( my_grid_type == GNOMONIC_ED ) {
     strcpy(projection, "cube_gnomonic");
     strcpy(conformal, "FALSE");
     if( do_schmidt || do_cube_transform ) {
       if( present_stretch_factor == 0 || present_target_lon == 0 || present_target_lat == 0 )
-	mpp_error("make_hgrid: grid type is 'gnomonic_ed, --stretch_factor, --target_lon "
-		  "and --target_lat must be set when --do_schmidt or --do_cube_transform is set");
+        mpp_error("make_hgrid: grid type is 'gnomonic_ed, --stretch_factor, --target_lon "
+                  "and --target_lat must be set when --do_schmidt or --do_cube_transform is set");
     }
 
     //if(nest_grids >= 1) {
     for (n=0; n < nest_grids; n++) {
 
       if(refine_ratio[n] == 0) mpp_error("make_hgrid: --refine_ratio must be set when --nest_grids is set");
       if(parent_tile[n] == 0 && mpp_pe()==mpp_root_pe()) {
-	fprintf(stderr,"NOTE from make_hgrid: parent_tile is 0, the output grid will have resolution refine_ration*nlon");
+        fprintf(stderr,"NOTE from make_hgrid: parent_tile is 0, the output grid will have resolution refine_ration*nlon\n");
+        //ntiles++;//Only the six faces may be needed for output, but this is to prevent invalid mem access //MZ2
```

Comment:
Is this a debugging comment line that should be removed?

Reply:
Ooops! Will remove.